source/ways-of-working/troubleshooting/index.html.md.erb

```diff
@@ -111,6 +113,106 @@ weight: 99
    > Always check __why__ your release or pod has failed in the first instance.
    > Although you may have permissions to delete a helm release or pod in a non-production environment, use this privilege wisely as you could be _hiding a potential bug_ which could also _occur in production_. 
 
+###
+> Please see [V1 docs](#flux-v1) for troubleshooting an issue with resources managed through Flux V1.
+
+### Latest image is not updated in cluster
+
+- Start with checking [cnp-flux-config](https://github.com/hmcts/cnp-flux-config) to make sure flux has updated/ committed the image. 
+- If image hasn't been committed to Github, see [ Flux did not commit latest image to Github](#flux-did-not-commit-latest-image-to-github).
+- If flux has committed the new image to Github, check if the `HelmRelease` has been updated by Flux. Run below command and check that the image tag has been updated in the output
+
+    ```shell
+    kubectl get helmreleases.helm.toolkit.fluxcd.io -n <your-namespace> <your-helm-release-name> -o yaml
+    ```
+- If Image is not updated in above, [Change in git is not applied to cluster](#change-in-git-is-not-applied-to-cluster).
```

Comment:
Should be pretty easy to poke it from Jenkins right?

Reply:
do u mean through a incoming notification in future? yes.

Reply:
Yes